### PR TITLE
feat: Showing suggested prompts in the Ai window based on the entities available in the app

### LINKED ISSUE #26691: Show suggested prompts in the Ai window based on the entities available in the app

### PR DESCRIPTION
**Pull request by Druthi**
## Description
feat: Showing suggested prompts in the Ai window based on the entities available in the app

## This PR fixes issues
Fixes #26691 

#### Type of change
- New feature (non-breaking change which adds functionality)

>
>
>
- [ ] Manual
- [ ] JUnit
- [ ] Jest
- [ ] Cypress
>
>
#### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#speedbreakers-) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#areas-of-interest-)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [ ] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

File: app/client/src/utils/storage.ts [hunk 1]
```diff
@@ -28,6 +28,7 @@ export const STORAGE_KEYS: {
   FEATURE_WALKTHROUGH: "FEATURE_WALKTHROUGH",
   USER_SIGN_UP: "USER_SIGN_UP",
   VERSION_UPDATE_STATE: "VERSION_UPDATE_STATE",
+  AI_RECENT_QUERIES: "AI_RECENT_QUERIES",
```
Comment (dvj1988) [inline app/client/src/utils/storage.ts:31]: This is merged to release. Please update with latest release branch.

File: app/client/src/selectors/navigationSelectors.ts [hunk 2]
```diff
@@ -119,6 +121,7 @@ export const getEntitiesForNavigation = createSelector(
         type: ENTITY_TYPE.WIDGET,
         url: widgetURL({ pageId, selectedWidgets: [widget.widgetId] }),
         children: result?.childNavData || {},
+        widgetType: widget.type,
```
Comment (dvj1988) [inline app/client/src/selectors/navigationSelectors.ts:124]: There is a key called `value` that is added to the type `NavigationData` but not passed here while calling `createNavData`. Do we need to pass it here?

**Review by dvj1988 — COMMENTED** [review]
Some minor comments. Rest looks good to me.

**Review by arunvjn — DISMISSED** [review]

**Review by dvj1988 — APPROVED** [review]

**Pull request merged**